src/components/views/dialogs/AddressPickerDialog.js

```diff
@@ -678,7 +678,7 @@ module.exports = createReactClass({
         }
 
         let identityServer;
-        if (this.props.pickerType === 'user' && !this.state.validAddressTypes.includes('email')) {
+        if (this.props.pickerType === 'user' && this.props.validAddressTypes.includes('email')) {
```

Comment:
For the case of email invites to a room without an IS at first, the props _will not_ include email, so this fix doesn't seem correct to me...

Maybe one way to handle the issue is changing `RoomInvite.js` to always add `email` to the `validAddressTypes` in props, but when computing initial state, **remove** email if we don't have an IS. Then finally, this condition could become:

```suggestion
        if (this.props.pickerType === 'user' && !this.state.validAddressTypes.includes('email') && this.props.validAddressTypes.includes('email')) {
```

So in effect, the props would specify the maximal set of address types, while the state would be what's currently accepted at the moment. (Probably would be nice to have a comment to that effect as well.)